caldav/caldav.go

```diff
@@ -67,8 +67,10 @@ type Calendar struct {
 	Path                  string
 	Name                  string
 	Description           string
+	Color                 string
 	MaxResourceSize       int64
 	SupportedComponentSet []string
+	Timezone              string
```

Comment:
This should probably be a parsed `*ical.Calendar`?

We should probably check that the parsed value contains exactly one `VTIMEZONE` component?